@ralph-lange @pablogs9 

update: 

With apt-get I could install osrf-testing-tools-cpp in /opt/ros/DISTRO in the ci.yml file configuration and the package is found in the build step.
However, now test_msgs is not found. The repository, which contains this package 
https://github.com/ros2/rcl_interfaces
is installed in my pc. I setup locally on my machine:

```C
sudo apt-get install ros-eloquent-osrf-testing-tools-cpp
sudo apt-get install ros-eloquent-rcl-interfaces
source /opt/ros/eloquent/setup.bash
mkdir -p ~/ros_ws/src
cd ~/ros_ws/src
git clone https://github.com/micro-ROS/rclc.git
cd ~/ros_ws; colcon_build --packages-up-to rclc
```
But here also the test_msgs could not be found. And indeed they are not part of /opt/ros/eloqent/share/rcl_interfaces/.


```C
CMake Error at /opt/ros/eloquent/share/ament_cmake_target_dependencies/cmake/ament_target_dependencies.cmake:62 (message):
  ament_target_dependencies() the passed package name 'test_msgs' was not
  found before
Call Stack (most recent call first):
  CMakeLists.txt:117 (ament_target_dependencies)


-- Configuring incomplete, errors occurred!
See also "/home/jst3si/test_ws_ci/build/rclc/CMakeFiles/CMakeOutput.log".
See also "/home/jst3si/test_ws_ci/build/rclc/CMakeFiles/CMakeError.log".
--- stderr: rclc
CMake Error at /opt/ros/eloquent/share/ament_cmake_target_dependencies/cmake/ament_target_dependencies.cmake:62 (message):
  ament_target_dependencies() the passed package name 'test_msgs' was not
  found before
```

Do you have any idea, how to resolve this?
Is this intended? Do I have to clone and re-build such repos?

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -40,7 +40,7 @@ jobs:
     - name : Download and install rclc-dependencies
       run: |
         apt-get install ros-${{ matrix.ros_distribution }}-osrf-testing-tools-cpp
-        apt-get install ros-${{ matrix.ros_distribution }}-rcl-interfaces
+        apt-get install ros-${{ matrix.ros_distribution }}-test-msgs
         apt-get install ros-${{ matrix.ros_distribution }}-test-interface-files
 
     #- name : Clone rclc-dependencies

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -41,7 +41,6 @@ jobs:
       run: |
         apt-get install ros-${{ matrix.ros_distribution }}-osrf-testing-tools-cpp
         apt-get install ros-${{ matrix.ros_distribution }}-test-msgs
-        apt-get install ros-${{ matrix.ros_distribution }}-test-interface-files
 
     - uses : ros-tooling/action-ros-ci@0.1.0
       with:

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -43,12 +43,6 @@ jobs:
         apt-get install ros-${{ matrix.ros_distribution }}-test-msgs
         apt-get install ros-${{ matrix.ros_distribution }}-test-interface-files
 
-    #- name : Clone rclc-dependencies
-    #  run: |
-    #    git clone https://github.com/osrf/osrf_testing_tools_cpp src/osrf_testing_tools_cpp
-    #    git clone -b ${{ matrix.ros_distribution }} https://github.com/ros2/rcl_interfaces.git src/rcl_interfaces
-    #    git clone -b ${{ matrix.ros_distribution }} https://github.com/ros2/test_interface_files.git src/test_interface_files
-
     - uses : ros-tooling/action-ros-ci@0.1.0
       with:
         package-name: "rclc rclc_examples rclc_lifecycle"

